app/src/main/java/com/example/sopt_assignment_dabin/SignInActivity.kt

```diff
@@ -0,0 +1,98 @@
+package com.example.sopt_assignment_dabin
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.animation.Animation
+import android.view.animation.AnimationUtils
+import android.widget.Toast
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import com.example.sopt_assignment_dabin.databinding.ActivitySignInBinding
+
+class SignInActivity : AppCompatActivity() {
+    private lateinit var getResultText: ActivityResultLauncher<Intent>  //회원가입 데이터 리턴받을 때 사용
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val binding = ActivitySignInBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        //fade in&fade out 애니메이션
+        fun setAnimation() {
+            val animation = AnimationUtils.loadAnimation(this, R.anim.fade_in)
+            val animation2 = AnimationUtils.loadAnimation(this, R.anim.fade_out)
+            binding.tvTitle.startAnimation(animation2)
+            binding.tvTitle.startAnimation(animation)
+        }
+        setAnimation()
+
+        //모든 데이터가 입력되었는지 확인
+        fun isEtIdEmpty(): Boolean {
+            return binding.etIdIn.text.isNullOrEmpty()
+        }
+
+        fun isEtPassword(): Boolean {
+            return binding.etPassIn.text.isNullOrEmpty()
+        }
+
+        fun isAllEditTextEmpty(): Boolean {
+            return isEtIdEmpty() || isEtPassword()
+        }
+
+
+        //모든 데이터가 입력되었을때 로그인 버튼 색깔 바뀌게하기->addTextChangedListener 사용
+        binding.etPassIn.addTextChangedListener(object : TextWatcher {
+            @SuppressLint("ResourceAsColor")
+            override fun afterTextChanged(p0: Editable?) {
+            }
+
+            override fun beforeTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+            }
+
+            override fun onTextChanged(p0: CharSequence?, p1: Int, p2: Int, p3: Int) {
+                if (isEtIdEmpty() != true && isEtPassword() != true && p0.toString() != "") {
+                    binding.bvLogin.setBackgroundResource(R.drawable.login_background2)
+                }
+            }
+        })
+
+       //로그인 버튼 눌렀을때 데이터 입력에 따른 분기 이벤트
+        binding.bvLogin.setOnClickListener {
+            if (isAllEditTextEmpty() == true) {
```

Comment:
```suggestion
            if (isAllEditTextEmpty()) {
```
이렇게 작성하셔도 괜찮을 것 같습니다~!!